src/PSRule.Rules.Azure/rules/Azure.ACR.Rule.ps1

```diff
@@ -57,6 +57,12 @@ Rule 'Azure.ACR.SoftDelete' -Ref 'AZR-000310' -Type 'Microsoft.ContainerRegistry
     $Assert.HasFieldValue($TargetObject, 'properties.policies.softDeletePolicy.retentionDays').Reason($LocalizedData.ACRSoftDeletePolicyRetention, $TargetObject.name)
 }
 
+# Synopsis: Disable export of artifacts from container registries.
+Rule 'Azure.ACR.ExportPolicy' -Ref 'AZR-000495' -Type 'Microsoft.ContainerRegistry/registries' -Tag @{ release = 'GA'; ruleSet = '2025_09'; 'Azure.WAF/pillar' = 'Security'; } -Labels @{ 'Azure.MCSB.v1/control' = @('DP-2') } {
+    # Check that export policy is disabled
+    $Assert.HasFieldValue($TargetObject, 'properties.policies.exportPolicy.status', 'disabled').Reason($LocalizedData.ACRExportPolicyDisabled, $TargetObject.name)
+}
+
```

Comment:
The rule in the file can be removed. You've already added it to. `Azure.ACR.Rule.yaml`.